src/cardStyles/stars.css

```diff
@@ -0,0 +1,19 @@
+.star {
+  display: flex;
+}
+
+.star__stars {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star--4 :nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);

```

Comment:
The previous comment is not fixed. Add styles for other modifiers (stars--1, stars--2 etc.). User should be able to change the amount of stars by changing the modifier
![image](https://user-images.githubusercontent.com/96409563/224533862-e6462226-e88e-422b-b3da-e04bfe963303.png)